app/javascript/components/CountrySelectionModal.tsx

```diff
@@ -54,7 +54,7 @@ export const CountrySelectionModal = ({ country: initialCountry, countries }: Pr
           if (referrer && new URL(referrer).origin === window.location.origin) {
```

Comment:
Inertia does update the history's state, so we _can_ use `window.history.back();`, but why is _this_ line back now? I thought we don't have access to `document.referrer` and that you needed to store the previous URL manually in order to perform that conditional check?

Can you please clarify and explain your changes here.